### PR TITLE
Updating to RC of OTel

### PR DESCRIPTION
**Pull request by samsp-msft**
Using the new UseOtlpExporter simplified hookup
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/3316)

### CODE REVIEW

**Review by sebastienros — APPROVED** [review]

File: src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/AspireApplication.1.ServiceDefaults.csproj [hunk 1]
```diff
@@ -12,8 +12,8 @@
 
     <PackageReference Include="Microsoft.Extensions.Http.Resilience" Version="!!REPLACE_WITH_EXTENSIONS_VERSION!!" />
     <PackageReference Include="Microsoft.Extensions.ServiceDiscovery" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
-    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.7.0" />
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.7.0" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.8.0-rc.1" />
```
[the remark this answers is not in the document]
Comment (samsp-msft) [reply src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/AspireApplication.1.ServiceDefaults.csproj:15]: Yes, the goal is to get them to stable. This is to validate that all that we need is in there.

**Comment by sebastienros** [comment]
Maybe let's also wait for 1.8.0 which should ship today (as Eric mentioned).

File: src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/Extensions.cs [hunk 2]
```diff
@@ -67,13 +67,10 @@ public static IHostApplicationBuilder ConfigureOpenTelemetry(this IHostApplicati
 
     private static IHostApplicationBuilder AddOpenTelemetryExporters(this IHostApplicationBuilder builder)
     {
-        var useOtlpExporter = !string.IsNullOrWhiteSpace(builder.Configuration["OTEL_EXPORTER_OTLP_ENDPOINT"]);
-
-        if (useOtlpExporter)
+        if (!string.IsNullOrWhiteSpace(builder.Configuration["OTEL_EXPORTER_OTLP_ENDPOINT"]))
```
Comment (eerhardt) [inline src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/Extensions.cs:70]: Please update all the ServiceDefaults projects under the `playground` folder as well. That way we are using the same code there as well.

**Comment by samsp-msft** [comment]
duplicate of #3351

**Pull request closed without merging**